.github/actions/pmc/action.yml

```diff
@@ -0,0 +1,15 @@
+name: PMC
```

Comment:
>PMC [](http://example.com/codeflow?start=6&length=3)

Would be good to use the full name once somewhere if possible (in a comment maybe?) To say what PMC means

Reply:
Good point! the `action.yml` metadata allows us to add a top-level `description` of the custom action. I will add one explaining that this is for packages.microsoft.com.